app/controllers/v1/pacbio/tag_sets_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module V1
+  module Pacbio
+     # TagSetsController
```

Comment:
Rubocop currently shouting about the indentation of this line.

Reply:
done.